<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DiscBackup</class>
 <widget class="QDialog" name="DiscBackup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>558</width>
    <height>177</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Instant Backup</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QLabel" name="lblStatus">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>481</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Andale Mono</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Please choose a type of backup and click on the Quick Backup button</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="backupProgress">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>110</y>
     <width>351</width>
     <height>23</height>
    </rect>
   </property>
   <property name="maximum">
    <number>0</number>
   </property>
   <property name="value">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QPushButton" name="btnClose">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>140</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QComboBox" name="boxBackupType">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>30</y>
     <width>241</width>
     <height>27</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Backup to CD/DVD</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Backup to External Drive</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="btnQuickBackup">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>344</x>
     <y>30</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Quick Backup</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblTime">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>311</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Andale Mono</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
